examples/mysql-query-row-operation/mysql_query_row_operation.bal

```diff
@@ -0,0 +1,71 @@
+import ballerina/io;
+import ballerinax/mysql;
+import ballerina/sql;
+
+// Defines a record to load the query result.
+type Customer record {|
+    int customerId;
+    string lastName;
+    string firstName;
+    int registrationId;
+    float creditLimit;
+    string country;
+|};
+
+public function main() returns error? {
+    // Runs the prerequisite setup for the example.
+    check initialize();
+
+    // Initializes the MySQL client. The `mysqlClient` can be reused to access database throughout the application execution.
```

Comment:
```suggestion
    // Initializes the MySQL client. The `mysqlClient` can be reused to access the database throughout the application execution.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)